domain/src/unit-test/java/uk/gov/moj/sdt/domain/IndividualRequestTest.java

```diff
@@ -257,12 +258,12 @@ void testPopulateSdtRequestReference() {
     @Test
     @DisplayName("Test Request toString")
     void testIndividualRequestToString() {
-        individualRequest.setRequestType("Request Type");
-        assertTrue(individualRequest.toString().contains("Request Type"), "Object to string should be populated");
+        individualRequest.setRequestType(REQUEST_TYPE_MESSAGE);
+        assertTrue(individualRequest.toString().contains(REQUEST_TYPE_MESSAGE), "Object to string should be populated");
     }
 
     @Test
-    @DisplayName("Test Request Interneral System Error")
+    @DisplayName("Test Request Individual System Error")
```

Comment:
The third word in the DisplayName refers to what the test is testing rather than the type of the request.  For this reason it should be "Test Request Internal System Error".